e2e/ansible/playbooks/feature/ha/ha.yml

```diff
@@ -29,14 +29,12 @@
        # (Includes installing packages, deploying apps)#
        #################################################
 
-       - include: ha-prerequisites.yml 
+       - include: ha-prerequisites.yml
 
-       - name: Get $HOME of K8s master for kubernetes user
-         shell: source ~/.profile; echo $HOME
-         args:
-           executable: /bin/bash
-         register: result_kube_home
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+            ns: openebs
+            app: maya-api
```

Comment:
Since we are updating the helm to use stable charts, maya-api server is deprecated and now it will be openeb-api server. Can you make the changes accordingly?